contrib/storage-http/README.md

```diff
@@ -138,11 +140,26 @@ key2=value2"
 ```
 
 `postBodyLocation`:  If the API uses the `POST` method, you can send parameters in several different ways:
-* `query_string`:  Parameters from the query are pushed down to the query string.  Static parameters are pushed to the post body.
```

Comment:
If the user enables this feature, are they required to specify the destination of the supplied parameters?  What happens if they don't?

Reply:
In the enhanced mode, default position is `json_body`.